tests/test_bivariate_analysis.py

```diff
@@ -450,9 +451,9 @@ def test_imports_verbosity_low_different_subsection_verbosities():
 
 
 @pyarrow_parameterize
-def test_show(pyarrow_dtypes: bool):
+def test_show(pyarrow_dtypes: bool, test_df: pd.DataFrame):
```

Comment:
I don't think the `pyarrow_dtypes` is passed as an argument to the `test_df` fixture. I've tried adding a code that raises exception when `pyarrow_dtypes=True` in `test_df` fixture and all the test passed, i.e. `pyarrow_dtypes` was always `False`.

I have pandas `2.2.1` and `pyarrow` installed.

Reply:
Good catch! You are right, it wasn't working. Fixed according to the [pytest documentation](https://docs.pytest.org/en/8.0.x/how-to/fixtures.html#parametrizing-fixtures) and confirmed to work using the "insert exception" method.